doc/releases/release-notes-4.0.rst

```diff
@@ -334,7 +334,7 @@ Drivers and Sensors
   * Introduced stepper shell commands for controlling and configuring
     stepper motors with :kconfig:option:`CONFIG_STEPPER_SHELL`
   * Added support for ADI TMC5041 (:dtcompatible:`adi,tmc5041`)
-  * Added support for gpio-stepper-controller (:dtcompatible:`gpio-stepper-controller`)
+  * Added support for gpio-stepper-controller (:dtcompatible:`zephyr,gpio-steppers`)
```

Comment:
fwiw the use of plural is a bit awkward and I am having a hard time finding other bindings using plural form, especially controllers

Reply:
Yeah i'll simplify the driver and make it singular